styles.css

```diff
@@ -14,7 +14,6 @@
   z-index: 100;
 }
 
-/* Logo container */
 .logo-container {
   width: 100px;
   height: 100%;
```

Comment:
[Optional] It would be nice if you add a background color to your toolbar so that it is better visible on scroll and does not affect the design👍